scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -86,14 +89,20 @@
 for test_tuple in test_tuples:
     os.environ["ROX_TELEMETRY_STORAGE_KEY_V1"] = 'DISABLED'
     test_versions = f'{test_tuple.central_version}--{test_tuple.sensor_version}'
+
+    is_374_sensor = test_tuple.sensor_version.startswith('74')
```

Comment:
Sorry for nitpicking but `'74'` creates enough confusion already. How about separating `3` from `74` in the variable name so that it hints towards ACS versions more?

```suggestion
    is_3_74_sensor = test_tuple.sensor_version.startswith('74')
```

Reply:
Sounds good - I've also added a little comment describing what the expected version string looks like, to hopefully clarify the `startwith` use